worker/main.ts

```diff
@@ -0,0 +1,39 @@
+import { config } from 'dotenv';
+
+import { MoveContactJobWorker } from './jobs/move-contact/move-contact-job.worker';
+import { QUEUE_NAMES } from '../shared/queues';
+
+config();
+
+interface WorkerConfig {
+  queueName: string;
+  workerClass: new (queueName: string) => any;
+}
+
+// Define the configuration for workers
+const workerConfigs: WorkerConfig[] = [
```

Comment:
in general, i like the spirit of what you're doing here to make a "generic worker engine" ... but i'm not sure how much value it is adding.  do you have other workers in mind which is motivating this?

in general, i think it might just be best to keep this simple and have a concrete `move-contacts-worker`. i'd suggest we skip most of this `generic worker engine` stuff

no need for a generic interface `WorkerConfig`. no need to loop over a const array of 1. no need for `jobs/move-contacts` because there is just one.  i like the `worker.ts` abstract stuff which handles some of the generic flow; but I don't get why there are two files for `move-contact-job.process` and `move-contact-job.worker`?

Reply:
Yes, thats was kind moving by separating concerns and make it easy to add a new worker in a need.
But you are right since we do not have any other worker in mind. We can keep it simpler and drop the generic stuff.